plaidml/bridge/openvino/ops/non_max_suppression.cpp

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "plaidml_ops.hpp"
+#include "plaidml_util.hpp"
+
+#include "ngraph/opsets/opset5.hpp"
+
+#include "plaidml/op/op.h"
+
+using namespace plaidml;  // NOLINT[build/namespaces]
+using ngraph::opset5::NonMaxSuppression;
+
+namespace PlaidMLPlugin {
+
+void registerNonMaxSuppression() {
+  registerOp("NonMaxSuppression", [](const Context& ctx) {
+    auto* layer = ngraph::as_type<NonMaxSuppression>(ctx.layer);
+    // OPSET5 provides multiple templates, follow setup() choice now.
+    IE_ASSERT(ctx.operands.size() == 6);
+    auto Boxes = ctx.operands.at(0);
+    auto Scores = ctx.operands.at(1);
+    int max_output_boxes_per_class = layer->max_boxes_output_from_input();
+    auto IOU_threshold = ctx.operands.at(3);
+    auto Score_threshold = ctx.operands.at(4);
+    float soft_nms_sigma = layer->soft_nms_sigma_from_input();
```

Comment:
This approach won't work for the non-constant non-zero tensor case. But, I think it's fine to not support that case for now and come back to a solution later with `is_soft_nms_sigma_constant_and_default` once we know of a use case that requires it.

Reply:
The function is_soft_nms_sigma_constant_and_default() will also check if the sigma value is 0, we can support the non-zero value so it is not appropriate now.  I changed soft_nms_sigma to float, but as you can see, we can support it as tensor in the former version.  The only one must be constant tensor is max_output_boxes_per_class now.  I have add verifier for both  max_output_boxes_per_class and soft_nms_sigma now.